src/components/structures/RoomView.js

```diff
@@ -897,18 +897,9 @@ module.exports = React.createClass({
         ev.stopPropagation();
         ev.preventDefault();
 
-        ev.dataTransfer.dropEffect = 'none';
```

Comment:
Don't we still need this to maintain the state?

Reply:
Not as far as I know. "none" doesn't mean "don't show the overlay", it means whatever you are dragging can't be dropped. https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect

Reply:
onDragLeaveOrEnd sets `draggingFile: false` so the overlay goes away

Reply:
oh hold on

Reply:
oh yeah this is right, if we put that in you loose the + sign by the pointer